Jenkinsfile

```diff
@@ -38,6 +39,11 @@ pipeline {
         dir("${BASE_DIR}"){
           loadConfigEnvVars()
         }
+        script {
```

Comment:
We created `whenTrue` and `whenFalse`steps to avoid to use the `script{ if() {} }` pattern 

```
          whenTrue(${params.debug}){
            dumpFilteredEnvironment()
          }
```

Reply:
Ah, good to know. Thanks! Fixed. :)